app/tests/api/test_email_status.py

```
@@ -0,0 +1,23 @@
+from fastapi.testclient import TestClient
+from app.main import app
+
+def test_email_status_endpoint(monkeypatch):
```

Ruff (0.15.1)

[warning] Unused function argument: `monkeypatch`

(ARG001)